app/controllers/admin/settings/virus_scanning_settings_controller.rb

```diff
@@ -55,6 +56,12 @@ def require_ee
       redirect_to action: :show
     end
 
+    def check_available
+      return if Setting.antivirus_scan_available?
+
+      render_404
+    end
+
```

Comment:
The same is not done for `admin/quarantined_attachments`. I don't think it a big problem since the tab is hidden from the user but it is inconsistent.

Reply:
Good point, changed 👍